server/src/main/resources/db/V26__add_uuid_cast.sql

```diff
@@ -0,0 +1,5 @@
+DROP CAST IF EXISTS (varchar AS uuid);
+
+CREATE CAST (varchar AS uuid)
+  WITH INOUT
+  AS IMPLICIT;
```

Comment:
And if you are casting on the query...you don't need this migration, right?

I have to admit...while this migration might make some things easier, It feels like we are hiding an issue from future contributors to this project.

Reply:
I tried casting on the query and it throws an error that explicit cast is needed. This is the only way I was able to make it work.

Reply:
how did you try casting on the query? what did that query look like exactly?

Reply:
I think I tried something like `WHERE (:checkinId as text IS NULL OR item.checkinid = :checkinId)`

Reply:
The way I got it to work on the draft PR was to simply use a String when making the database call. Open debate to be had on if using the String is better or worse than having to explicitly cast the null case in the query.